pkg/frontend/v2/frontend_scheduler_worker.go

```diff
@@ -266,18 +266,15 @@ func (w *frontendSchedulerWorker) stop() {
 }
 
 func (w *frontendSchedulerWorker) runOne(ctx context.Context, client schedulerpb.SchedulerForFrontendClient) {
-	backoffConfig := backoff.Config{
-		MinBackoff: 250 * time.Millisecond,
-		MaxBackoff: 2 * time.Second,
-	}
+	// attemptLoop returns false iff it couldn't properly start or end the loop
```

Comment:
Not sure if this comment is correct: "end the loop"... what about:

```suggestion
	// attemptLoop returns false if there was any error with forwarding requests to scheduler.
```